### PR TITLE
Update week_three.md

### PR DESCRIPTION
**Pull request by Robbie-Smith**

### CODE REVIEW

File: module-2/backend/week_three.md [hunk 1]
```diff
@@ -10,14 +10,31 @@ Note: When you're done, submit a PR with a reflection in the comments about how
 ### Questions
 
 1. What is the entry at the command line to create a new rails app?
+  * rails -new
```
Comment (s-espinosa) [inline module-2/backend/week_three.md]: Don't need the `-`

File: module-2/backend/week_three.md [hunk 2]
```diff
@@ -10,14 +10,31 @@ Note: When you're done, submit a PR with a reflection in the comments about how
...
 2. What do Models generally inherit from in rails?
+  * ApplicationRecord::Base or ActiveRecord::Base
 3. What do Controllers generally inherit from in a rails project?
+  * ApplicationController
 4. How would I create a route if I wanted to see a specific horse in my routes fitle assuming I'm sticking to standard conventions and that I didn't want other CRUD functionality?
+  * resources :horses, only:[:show]
 5. What rake task is useful when looking at routes, and what information does it give you?
+  * rake -T
```
Comment (s-espinosa) [inline module-2/backend/week_three.md]: That's certainly helpful, but when looking at routes `rake routes` is usually my go to.

File: module-2/backend/week_three.md [hunk 3]
```diff
@@ -10,14 +10,31 @@ Note: When you're done, submit a PR with a reflection in the comments about how
...
 6. What is an example of a route helper? When would you use them?
+  * link_to "Stuff", stuff_path
+  * To route to things.
 7. What's the difference between what `_url` and `_path` return when combined with a routes prefix?
+  * path is just the path, and url is the complete url of the website.
```
Comment (s-espinosa) [inline module-2/backend/week_three.md]: 👍

File: module-2/backend/week_three.md [hunk 4]
```diff
@@ -10,14 +10,31 @@ Note: When you're done, submit a PR with a reflection in the comments about how
...
 8. What are strong params and why are the necessary?
+  * the limit access to the things the user can fill in. The prevent the user from breaking/doing things they are not supposed to.
 9. What role does `form_for` play in helping us create our forms?
+  * it is a form helper for rails.
 10. How does `form_for` know where to submit the user's input?
+  * it uses the f. tags. like <%= f.submit>
```
Comment (s-espinosa) [inline module-2/backend/week_three.md]: In terms of knowing *where* to submit the information in the form, it takes a look at the argument (e.g. `@horse` in the example below), and determines if it's been saved to the database or not. If it hasn't, it creates a form that will submit a POST request to `/horses`. If it has, it creates a form that will submit a PATCH request to `/horses/:id`.

File: module-2/backend/week_three.md (hunk 2 above)
Comment (Robbie-Smith) [inline module-2/backend/week_three.md]: Thanks! I misread the question.

File: module-2/backend/week_three.md (hunk 1 above)
Comment (Robbie-Smith) [inline module-2/backend/week_three.md]: Thank you

File: module-2/backend/week_three.md (hunk 4 above)
Comment (Robbie-Smith) [inline module-2/backend/week_three.md]: Thank you!

**Pull request closed without merging**